nvdaHelper/remote/textFromIAccessible.h

```diff
@@ -16,6 +16,16 @@ This license can be found at:
 #define IA2TEXTFROMIACCESSIBLE_H
 struct IAccessible2;
 
+/* getTextFromIAccessible
+* Summarizes the text for an IAccessible
+* @param textBuf: An out-param, the string to fill with the text from the IAccessible
+* @param pacc2: The IAccessible2 object to get the text from.
+* @param useNewText: Only valid during an in-process winEvent callback (ie text changed). Specifies
+*    to use the new text (ie post change) should be used instead of the old text.

```

Comment:
```suggestion
*    to use the new text (ie only the inserted/changed text) should be used instead of all the current text in the Accessible. 
```

Reply:
Is current really the right description? Surely this the event is saying the text has already changed.

Reply:
I can see why current might be ambiguous, but I am struggling to find a better word. Either way though, "old text" is not correct.
Perhaps I can illustrate with an example and then you can find wording you are happy with.
* Let there be an Accessible object whose text is "abcde", I.e. fetched from IAccessibleText::text.
* Some new text is inserted, and therefore a textInserted event is fired.
* IAccessibleText::text now produces "abxxcde" (I.e. "xx" was inserted after "ab").
* Within the context of the textInserted event, IAccessibleText::newText is "xx". 
So if this function was called within the textInserted event, it would fetch "abxxcde" if useNewText is false, and "xx" if useNewText is true.

Reply:
Oh I see. Yes, that helps.
It looks like this only applies to the events:
- `IA2_EVENT_TEXT_INSERTED`
- `IA2_EVENT_TEXT_REMOVED`
- `IA2_EVENT_TEXT_UPDATED`.

In the event handlers for those events, `IAccessibleText::text` will reflect the fully realized modification.
`IAccessibleText::newText` is used to determine the additions to the overall text in the case of `IA2_EVENT_TEXT_INSERTED` and `IA2_EVENT_TEXT_UPDATED` and `IAccessibleText::oldText` is used to determine the removals from the overall text in the case of `IA2_EVENT_TEXT_REMOVED` and `IA2_EVENT_TEXT_UPDATED`

When `useNewText` is given, then the fetched text is only the text added, ie we only fetch `IAccessibleText::newText`